planner/core/planbuilder.go

```diff
@@ -833,6 +856,10 @@ const (
 )
 
 func (b *PlanBuilder) buildAnalyze(as *ast.AnalyzeTableStmt) (Plan, error) {
+	// If enable fast analyze, the storage must be tikv.
```

Comment:
Should we also support it in mock tikv? Or we cannot write unit tests.

Reply:
I'll investigate it.

Reply:
Does mock tikv have RegionCache and client? I feel puzzled about how to support in mock tikv.

Reply:
Yes, you can take a look at `mocktikv.NewTiKVAndPDClient`.

Reply:
Already supported. There no code modify here, because the mock tikv is the tikv storage. But we need add `DebugGetProperies` RPC handle in `RPCClient`.

Reply:
But it does matter if we use another handle or not? As long as it can process that specific request and returns the right result.